lib/Partitioner/PartitionUtils.cpp

```diff
@@ -0,0 +1,130 @@
+
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Partitioner/PartitionUtils.h"
+
+#include <iostream>
+#include <set>
+
+namespace glow {
+
+/// Given \p nodes, return a list of nodes who use any node in this set.
```

Comment:
This is really any nodes who use this set that are not in this set right?

Reply:
Yes :)